### ISSUE TITLE
Finetune issue

### ISSUE THREAD

**Issue by YaosuLee** [giannisdaras/ambient-diffusion#7]
Hi, thank you for your work. I have a question: would it be possible to fine-tune DeepFloyd/IF-I-M-v1.0 instead of DeepFloyd/IF-I-XL-v1.0? When I try training with the smaller pretrained model (IF-I-M-v1.0), it struggles to reconstruct the image and remove the corrupted mask (both dust & fixed-box). Is this fine-tuning possible, or are there specific hyperparameters I should focus on to improve the results?
